test/integration/mount_start_test.go

```diff
@@ -123,26 +142,47 @@ func validateMount(ctx context.Context, t *testing.T, profile string) {
 	}
 
 	args = sshArgs
-	args = append(args, "mount", "|", "grep", "9p")
+	args = append(args, "findmnt", "--json", guestPath)
 	rr, err = Run(t, exec.CommandContext(ctx, Target(), args...))
 	if err != nil {
 		t.Fatalf("failed to get mount information: %v", err)
 	}
 
+	result, err := findmnt.ParseOutput(rr.Stdout.Bytes())
+	if err != nil {
+		t.Fatalf("failed to parse command output %q: %s", rr.Command(), rr.Output())
+	}
+
+	if len(result.Filesystems) == 0 {
+		t.Fatalf("no filesystems found")
+	}
+
+	fs := result.Filesystems[0]
+	if fs.FSType == "virtiofs" {
```

Comment:
better to make this a type like driver names

Reply:
Using constants.FSTypeVirtiofs